<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EZ-20170527FOCB" name="test.publictest.InitTest" tests="3" failures="0" timestamp="21 九月 2017 10:25:03 GMT" time="20.787" errors="3">
  <testcase name="init" time="9.441" classname="test.publictest.InitTest">
    <error type="org.springframework.beans.factory.NoSuchBeanDefinitionException" message="No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: ">
      <![CDATA[org.springframework.beans.factory.NoSuchBeanDefinitionException: No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: 
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:271)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1101)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:463)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:420)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:386)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at test.publictest.InitTest.init(InitTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- init -->
  <testcase name="init" time="6.122" classname="test.publictest.InitTest">
    <error type="org.springframework.beans.factory.NoSuchBeanDefinitionException" message="No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: ">
      <![CDATA[org.springframework.beans.factory.NoSuchBeanDefinitionException: No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: 
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:271)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1101)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:463)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:420)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:386)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at test.publictest.InitTest.init(InitTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- init -->
  <testcase name="init" time="5.224" classname="test.publictest.InitTest">
    <error type="org.springframework.beans.factory.NoSuchBeanDefinitionException" message="No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: ">
      <![CDATA[org.springframework.beans.factory.NoSuchBeanDefinitionException: No unique bean of type [page.publicpage.LoginPage] is defined: expected single bean but found 0: 
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:271)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1101)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:463)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:420)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at com.glodon.autotest.web.framework.settings.SettingUtil.parse(SettingUtil.java:386)
	at com.glodon.autotest.web.framework.settings.SettingUtil.read(SettingUtil.java:207)
	at com.glodon.autotest.web.framework.settings.SettingUtil.readFromClassPath(SettingUtil.java:159)
	at test.publictest.InitTest.init(InitTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- init -->
</testsuite> <!-- test.publictest.InitTest -->
